src/operation/abort_transaction/test.rs

```diff
@@ -0,0 +1,66 @@
+use pretty_assertions::assert_eq;
+
+use crate::{
+    bson::doc,
+    client::session::TransactionPin,
+    cmap::StreamDescription,
+    concern::{Acknowledgment, WriteConcern},
+    operation::{AbortTransaction, Operation},
+    selection_criteria::{ReadPreference::Primary, SelectionCriteria::ReadPreference},
+};
+
+#[cfg_attr(feature = "tokio-runtime", tokio::test)]
+#[cfg_attr(feature = "async-std-runtime", async_std::test)]
+async fn build() {
```

Comment:
It looks like these (and similar tests for other operations) don't need to be async, which means they can just be annotated with `#[test]` instead of the runtime test whatnots.

Reply:
Every test for all operations include these features—should I re-annotate them all with `#[test]` and remove the `async`?

Reply:
I hadn't noticed that there were a bunch of existing operation tests that were async!  I think to avoid churn the best way here would be to update the tests that you've added in this PR and then file a ticket with the greenerbuild tag to update the rest of them.

Reply:
Sounds good, filed [RUST-1102](https://jira.mongodb.org/browse/RUST-1102) to track this.